elements/bulbs-video/components/revealed.js

```diff
@@ -171,7 +176,7 @@ export default class Revealed extends React.Component {
     let type;
 
     // See docs (https://support.google.com/dfp_premium/answer/1068325?hl=en) for param info
-    baseUrl += '?sz=640x480';
+    baseUrl += `?sz=${this.props.creativeSize}`;
```

Comment:
So, you should either find a way to use the `creativeSize` you defined above in the `let` assignment or do what you're doing here but otherwise the assignment above isn't used.